components/footer/01-footer/footer.html

```diff
@@ -0,0 +1,81 @@
+<footer class="mzp-c-footer">
+  <div class="mzp-l-content">
+    {% block primary %}
+    <nav class="mzp-c-footer-primary">
+      <div class="mzp-c-footer-primary-logo">
+        <a href="https://www.mozilla.org/">Mozilla</a>
+      </div>
+      <div class="mzp-c-footer-sections">
+        {% for section in range(nb_sections) %}
+        <section class="mzp-c-footer-section">
+          <h5 class="mzp-c-footer-heading">
+            Topic
+          </h5>
+          <ul class="mzp-c-footer-list">
+            <li>
+              <a href="https://www.mozilla.org/firefox/new/">Link One</a>
+            </li>
+            <li>
+              <a href="https://www.mozilla.org/firefox/">Link Two</a>
+            </li>
+            <li>
+              <a href="https://www.mozilla.org/firefox/mobile/">Link Three</a>
+            </li>
+            <li>
+              <a href="https://www.mozilla.org/firefox/features/">Link Four</a>
+            </li>
+            <li>
+              <a href="https://www.mozilla.org/firefox/channel/desktop/">Long link just to mix things up and cause problems.</a>
+            </li>
+            <li>
+              <a href="https://www.mozilla.org/firefox/channel/desktop/">Link Six</a>
+            </li>
+          </ul>
+        </section>
+        {% endfor %}
+      </div>
+    </nav>
+    {% endblock %}
+
+    {% block secondary %}
+    <nav class="mzp-c-footer-secondary">
+      <div class="mzp-c-footer-language">
+        {% render '@language-switcher'  %}
+      </div>
+      <ul class="mzp-c-footer-links-social">
+        <li>
+          <a class="twitter" href="https://twitter.com/firefox">Twitter<span> (@firefox)</span></a>
+        </li>
+        <li>
+          <a class="youtube" href="https://www.youtube.com/firefoxchannel">YouTube<span> (firefoxchannel)</span></a>
+        </li>
+      </ul>
+      <div class="mzp-c-footer-legal">
+        <p class="mzp-c-footer-license">
+          Visit Mozilla Corporation’s not-for-profit parent, the <a href="https://foundation.mozilla.org/">Mozilla Foundation</a>.<br>
+          Portions of this content are ©1998–2018 by individual mozilla.org contributors.
+          Portions of this content are ©1998–2018 by individual mozilla.org contributors.
```

Comment:
This repetition seems like a mistake? Overactive paste? 
(And we might as well bump the date to 2022 even though it's just an example).